specs/Pills.md

```diff
@@ -0,0 +1,182 @@
+# [Pills]()
+
+## Background
+
+Pills should be used when representing an input, as a way to filter content, or to represent an attribute.
+
+## Prior Art
+
+- [Evergreen](https://evergreen.segment.com/components/badge-and-pill/)
+- [Lightning](https://www.lightningdesignsystem.com/components/pills/)
+- [Material UI](https://material-ui.com/components/chips/#chip)
+- [OpenUI Research](https://github.com/WICG/open-ui/pull/259)
+
+## Sample Code
+
+```jsx
+<Pill>Item 1</Pill>
```

Comment:
aren't we missing `<Pills>` in the example? The goal of this work should be to create both container and item components.

Reply:
Added